apps/system/js/app_window_manager.js

```diff
@@ -488,6 +490,13 @@
       var activeApp = this._activeApp;
       var detail = evt.detail;
       switch (evt.type) {
+        case 'hierarchytopmostwindowchanged':
```

Comment:
For hierarchytopmostwindowchanged I think so. but I am fined to change to windowopened/windowclosed